### PR TITLE
Cleanup - lower barrier to entry, more gradual difficulty

### PR DESCRIPTION
**Pull request by gavD**
Clean up code
Separate exercises into separate file
Simpler function names
The birds variable can be set to determine number of birds
Simplified some exercises
Added 'Things you've used' to the bottom of the page

### CODE REVIEW

File: js/jscene-iframe.js [hunk 1]
```diff
@@ -56,7 +58,7 @@ var testConditions = {
  *
  * @returns {null}
  */
-function setTimeOfDay(time) {
+function settime(time) {
```
Comment (jenkoian) [inline js/jscene-iframe.js:61]: Not using camelCase for this one?

File: docs/jscene-iframe.html [hunk 2]
```diff
@@ -387,7 +387,7 @@
               <div class="pilwrap ">
                 <a class="pilcrow" href="#section-17">&#182;</a>
               </div>
-              <p>Function to draw the moon. This functino is a wrapper of showBallInSky</p>
+              <p>Function to draw the moon. This functino is a wrapper of orb</p>
```
Comment (craigmarvelley) [inline docs/jscene-iframe.html]: Typo (functino). Is wrapper a term that kids will understand?

File: js/jscene-iframe.js (hunk 1 above)
Comment (gavD) [inline js/jscene-iframe.js:61]: no; mixed case was confusing kids

File: docs/jscene-iframe.html (hunk 2 above)
Comment (craigmarvelley) [inline docs/jscene-iframe.html]: Ah, is not for kids, is it?

Comment (gavD) [inline docs/jscene-iframe.html]: Almost certainly not. A lot of that copy needs rewriting...

File: js/jscene-iframe.js (hunk 1 above)
Comment (jenkoian) [inline js/jscene-iframe.js:61]: ah I see, what about https://github.com/boxuk/jscene/pull/3/files#L6R195 or is that only used privately?

Comment (gavD) [inline js/jscene-iframe.js:61]: yeah that's not exposed

Comment (jenkoian) [inline js/jscene-iframe.js:61]: :+1:

**Pull request merged**